### PR TITLE
add multi-column tests

### PR DESCRIPTION
**Pull request by birdstorm**
### What problem does this PR solve? 
add multi column tests

### What is changed and how it works?
tests

### Check List 

Tests 

 - Integration test

Side effects

 - Possible test performance regression

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/sql/BaseTestGenerationSpec.scala [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2017 PingCAP, Inc.
+ * Copyright 2019 PingCAP, Inc.
```
Comment (marsishandsome) [inline core/src/test/scala/org/apache/spark/sql/BaseTestGenerationSpec.scala]: what's the rule about when to update the `Copyright year`?

File: core/src/test/scala/org/apache/spark/sql/MysqlDataTypeSuite.scala [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /*
+ *
```
Comment (marsishandsome) [inline core/src/test/scala/org/apache/spark/sql/MysqlDataTypeSuite.scala]: should we add `license check` in scalafmt?

File: core/src/test/scala/org/apache/spark/sql/BaseTestGenerationSpec.scala (hunk 1 above)
Comment (birdstorm) [reply core/src/test/scala/org/apache/spark/sql/BaseTestGenerationSpec.scala]: I forgot to use the right copyright year when I copy-pasted it from another file.

File: core/src/test/scala/org/apache/spark/sql/MysqlDataTypeSuite.scala (hunk 2 above)
Comment (birdstorm) [reply core/src/test/scala/org/apache/spark/sql/MysqlDataTypeSuite.scala]: sure.

**Comment by birdstorm** [comment]
Also updates a graph of current tests(should be less messy than before:D) @marsishandsome @zhexuany 

image

**Pull request merged**